galite-core/src/main/kotlin/org/kopi/galite/util/Fax.kt

```diff
@@ -0,0 +1,660 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ * Copyright (c) 1990-2020 kopiRight Managed Solutions GmbH, Wien AT
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.kopi.galite.util
+
+import org.kopi.galite.base.Utils
+
+import java.io.BufferedReader
+import java.io.ByteArrayOutputStream
+import java.io.DataInputStream
+import java.io.DataOutputStream
+import java.io.IOException
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.io.PrintWriter
+import java.net.ConnectException
+import java.net.InetAddress
+import java.net.ServerSocket
+import java.net.Socket
+import java.util.StringTokenizer
+import java.util.Vector
+import java.util.zip.Deflater
+import java.util.zip.DeflaterOutputStream
+
+/**
+ * @deprecated  replaced by the class HylaFAXUtils
+ */
+class Fax(port: Int, host: String) {
+
+  fun login(uname: String): Int {
+    Utils.log("Fax", "login:$uname")
+    print("USER $uname\n")
+    clntOut.flush()
+    val answer = check(readLine())
+    if (answer == NEEDS_PASSWD) {
```

Comment:
can we remove answer and change this to 
  `  if (check(readLine()) == NEEDS_PASSWD) {`